package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@eppo/js-client-sdk-common",
-  "version": "4.7.0",
+  "version": "4.7.0-alpha.1",
```

```suggestion
  "version": "5.0.0-alpha.1",
```

removing some public methods that are used by customers (see below)

---

See my other comment — we are not removing used methods. All changes are for the new precomputed client, not the regular one